### ISSUE TITLE
[DOC] Integration of PPL into Data Explorer

### ISSUE THREAD

**Issue by seanneumann** [opensearch-project/documentation-website#5529]
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [X] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 We are creating a new Data Explorer experience that combines functionality of Discover and Observability:Logs. This also means we are integrating PPL into the query bar. We need user documentation to accompany this change. 

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
Related eng issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5310

**Comment by vagimeli** [comment]
Hi @mengweieric: This issue shows the feature is releasing for 2.12. **Please provide the draft PR no later than first RC date.** This SLA ensures we can move the documentation through the documentation and editorial review processes, including working with you to address any information gaps, rewrite content, and publish the documentation, at the time of release. Note that if the PR is provided after the first RC date, the documentation may not be published with the release.

The documentation manager (@hdhalter) has updated the writing roles and responsibilities. For a refresher about these roles and responsibilities, see the [Working with the Documentation team](https://w.amazon.com/bin/view/OpenSearch_Project/Documentation/DocProcesses/Working_with_docs/) wiki. You're welcome to reach out to me with any questions about supporting the writing tasks for your feature. Thank you!

**Comment by vagimeli** [comment]
@wbeckler Please let me know if this feature is on schedule for 2.12. Also, who is the dev I can work with to put up a PR? Thank you, @vagimeli

**Comment by mengweieric** [comment]
> > Hi @mengweieric: This issue shows the feature is releasing for 2.12. **Please provide the draft PR no later than first RC date.** This SLA ensures we can move the documentation through the documentation and editorial review processes, including working with you to address any information gaps, rewrite content, and publish the documentation, at the time of release. Note that if the PR is provided after the first RC date, the documentation may not be published with the release.
> > The documentation manager (@hdhalter) has updated the writing roles and responsibilities. For a refresher about these roles and responsibilities, see the [Working with the Documentation team](https://w.amazon.com/bin/view/OpenSearch_Project/Documentation/DocProcesses/Working_with_docs/) wiki. You're welcome to reach out to me with any questions about supporting the writing tasks for your feature. Thank you!
> 
> Hi @mengweieric: I'm following up on the status of this feature. Is it on track for 2.12? If so, has a documentation PR been created?

Hi @vagimeli, for 2.12, there will be no PPL integration for data explorer due to scope changes recently.

**Comment by hdhalter** [comment]
Thanks, @mengweieric ! I've removed the 2.12 label. Please let us know if this gets picked up again.

**Comment by hdhalter** [comment]
Hi @mengweieric - Is this targeting 2.13 by any chance? FYI, the release-train field still says, "2.12".

**Comment by mengweieric** [comment]
@hdhalter It will not be for 2.13, PPL integration is happening along with cohesion that is targeted for 2.14 based on the current planning.

**Comment by hdhalter** [comment]
Hi @mengweieric , can you please confirm that this is targeting 2.14?

**Comment by mengweieric** [comment]
Yes, this is currently in the 2.14 release road map, @kavilla is working on this.

**Comment by hdhalter** [comment]
> Yes, this is currently in the 2.14 release road map, @kavilla is working on this.

Thanks! Do you know who will be producing the doc PR?

**Comment by mengweieric** [comment]
> > Yes, this is currently in the 2.14 release road map, @kavilla is working on this.
> 
> Thanks! Do you know who will be producing the doc PR?

That I believe is TBD, may need to check with @wbeckler @anirudha on who should take it.

**Comment by hdhalter** [comment]
Hi @wbeckler and @anirudha - do you know who is producing the doc PR?

**Comment by ashwin-pc** [comment]
@kavilla can you help with this doc?

**Comment by hdhalter** [comment]
Doc PR in progress.

**Comment by hdhalter** [comment]
@kavilla - What is the status on the doc?

**Comment by hdhalter** [comment]
@kavilla , @ashwin-pc - Is this going out in 2.15?

**Comment by natebower** [comment]
Closing as stale
